docs/generate_extension_db.py

```diff
@@ -37,18 +38,33 @@ def GetExtensionMetadata(target):
   if not BUILDOZER_PATH:
     raise ExtensionDbError('Buildozer not found!')
   r = subprocess.run(
-      [BUILDOZER_PATH, '-stdout', 'print security_posture status undocumented', target],
+      [BUILDOZER_PATH, '-stdout', 'print security_posture status undocumented category', target],
       stdout=subprocess.PIPE,
       stderr=subprocess.PIPE)
-  security_posture, status, undocumented = r.stdout.decode('utf-8').strip().split(' ')
+  rout = r.stdout.decode('utf-8').strip().split(' ')
+  security_posture, status, undocumented = rout[:3]
+  categories = ' '.join(rout[3:])
   if IsMissing(security_posture):
     raise ExtensionDbError(
         'Missing security posture for %s.  Please make sure the target is an envoy_cc_extension and security_posture is set'
         % target)
+  # evaluate tuples/lists
+  # wrap strings in a list
+  # use empty list for missing (this should not be necessary)
+  categories = (
+    '[]'
+    if IsMissing(categories)
+    else categories)
+  categories = (
+    ast.literal_eval(categories)
+    if ('[' in categories
+        or '(' in categories)
+    else [categories])
```

Comment:
this duck typing + eval is not ideal, but im struggling to think of a way of using the bazel query and allowing cats to be strings or arrays